tests/wolfram_alpha_test.py

```diff
@@ -2,26 +2,31 @@
 import logging
 
 import vcr
-from errbot.backends.test import TestBot
+from errbot.backends.test import FullStackTest
 
 from plugins.wolfram_alpha import WolframAlpha
 
 my_vcr = vcr.VCR(match_on=['method', 'scheme', 'host', 'port', 'path'],
                  filter_query_parameters=['appid'])
 
 
-class WolframAlphaTest(unittest.TestCase):
+class WolframAlphaTest(FullStackTest):
 
-    def setUp(self):
-        self.testbot = TestBot(extra_plugin_dir='plugins',
-                               loglevel=logging.ERROR)
-        self.testbot.start()
+    def setUp(self,
+              extra_plugin_dir=None,
+              extra_test_file=None,
+              loglevel=logging.DEBUG,
+              extra_config=None):
+        super().setup(extra_plugin_dir='plugins',
```

Comment:
Use `setUp` and remove next line. Similarly in other cases

Reply:
I should die :persevere:

Reply:
It's okay, no need to die.